### ISSUE TITLE
Problem installing Git Credential Manager Core (Ubuntu 22.04.lts)

### ISSUE THREAD

**Issue by CornaSn** [upleveled/system-setup#56]
[https://github.com/upleveled/system-setup/blob/main/linux.md](https://github.com/upleveled/system-setup/blob/main/linux.md)
Tried this comment: 
```
curl -L https://aka.ms/gcm/linux-install-source.sh | sh
git-credential-manager-core configure
``` 
This leads to:

```
$ curl -L https://aka.ms/gcm/linux-install-source.sh | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  7233  100  7233    0     0  54705      0 --:--:-- --:--:-- --:--:-- 54705
This script will download, compile, and install Git Credential Manager to:

    /usr/local/bin

Git Credential Manager is licensed under the MIT License: https://aka.ms/gcm/license
Please answer yes or no.
Please answer yes or no.
Please answer yes or no.
Please answer yes or no.
Please answer yes or no.
Please answer yes or no.
Please answer yes or no.
Please answer yes or no.
sh: 169: Syntax error: "fi" unexpected (expecting ";;")
```
With no chance to answer anything. Therefore the next step also doesn't work

**Comment by karlhorky** [comment]
Interesting, these are [the official installation instructions](https://github.com/git-ecosystem/git-credential-manager/blob/release/docs/install.md#install-from-source-helper-script)

Maybe [the alternative installation method to install the `.dpkg`](https://github.com/git-ecosystem/git-credential-manager/blob/release/docs/install.md#install-2) would help?

**Comment by CornaSn** [comment]
Installing via package works.

But it seems you could install it via rpm so maybe this is a better approach:

[https://github.com/microsoft/Git-Credential-Manager-for-Mac-and-Linux/blob/master/Install.md]()
```
sudo rpm --install git-credential-manager-2.0.4-1.noarch.rpm
git-credential-manager install
```

**Comment by karlhorky** [comment]
The link you posted above is to the old archived repo, which was archived in 2022 (check the notice at the top of the page)

**Comment by karlhorky** [comment]
So I would say probably:

1. the issue with installing from source should be reported in their repo
2. in the meantime, before it's fixed, the `.deb` file installation can be used

**Comment by karlhorky** [comment]
@CornaSn any news here? Did you manage to get it installed with the `.deb` file?

**Comment by CornaSn** [comment]
Yes I installed it with the .deb file and it worked

**Comment by ProchaLu** [comment]
There are 2 open issues for this problem.
- https://github.com/git-ecosystem/git-credential-manager/issues/1597
- https://github.com/git-ecosystem/git-credential-manager/issues/1577

Apparently, in [this PR](https://github.com/git-ecosystem/git-credential-manager/pull/1469) it should be fixed, [released in version 2.5.0.](https://github.com/git-ecosystem/git-credential-manager/releases/tag/v2.5.0)

**Comment by ProchaLu** [comment]
Fixed in https://github.com/git-ecosystem/git-credential-manager/pull/1469 and [released in version 2.5.0.](https://github.com/git-ecosystem/git-credential-manager/releases/tag/v2.5.0)
